enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/helper/ErrorHandler.java

```diff
@@ -17,17 +17,37 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.causalclustering.discovery;
+package org.neo4j.causalclustering.helper;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 
+import org.neo4j.function.ThrowingAction;
+
+
 public class ErrorHandler implements AutoCloseable
 {
     private final List<Throwable> throwables = new ArrayList<>();
     private final String message;
 
+    public static void certainOperations( String description, ThrowingRunnable... actions ) throws RuntimeException
```

Comment:
The "certain" part doesn't semantically make sense to me.

Reply:
just "runAll"?